src/roadmap/releases.md

```diff
@@ -133,11 +133,14 @@ If the component has a correlating [Kubernetes Helm][] chart,
 this chart will also be packaged, signed and uploaded to its production chart repo.  Please
 verify it can be fetched (and verified):
 
+!!! note
+    Verification of chart provenance is [currently broken](https://github.com/deis/workflow/issues/822).
```

Comment:
You're right--"broken" implies that we want to fix it ASAP, which I *think* is true. This particular document is public, but not really user-facing (it's our instructions to ourselves about how to do a Workflow release), so I think it's ok to say "broken" here.